### PR TITLE
[MSYS-729]: SSH prompt fix windows

### PR DESCRIPTION
**Pull request by Nimesh-Msys**
Fix for SSH Prompt includes following changes:
1: To display the passphrase prompt, we need to use Stdio (Stdio::inherit())
2: On windows we are required to use SSH agent, hence we rebuild the command surrounded with SSH agent's creation and deletion; Then only the entered passphrase is significant

Signed-off-by: Nimesh Patni <nimesh.patni@msystechnologies.com>

### CODE REVIEW

**Comment by thommay** [comment]
This supersedes #44.

File: src/delivery/git/mod.rs [hunk 1]
```diff
@@ -96,8 +96,25 @@ pub fn git_command<P>(args: &[&str], c: &P) -> Result<GitResult, DeliveryError>
...
+      let agent_kill = "; ssh-agent -k";
+      let cmd_args = args.join(" ");
+      let command_ars = &[ &[agent_setup, &cmd_args, agent_kill].join(" ") ];
+      command.args(command_ars);
+    } else {
+      command.args(args);
+    }
+
     command.current_dir(cwd);
+    command.stderr(Stdio::inherit());
```
Comment (thommay) [inline src/delivery/git/mod.rs]: This should only be on windows, and so needs guarding, correct?

**Comment by Vasu1105** [comment]
@btm @thommay Could you please review this and merge if looks goods pending from long time.

**Comment by btm** [comment]
travis look broken on master too :(

https://travis-ci.org/chef/delivery-cli/jobs/381629459

**Pull request merged**